src/nvim/lua/executor.c

```diff
@@ -176,6 +178,11 @@ static int nlua_state_init(lua_State *const lstate) FUNC_ATTR_NONNULL_ALL
   lua_pushcfunction(lstate, &nlua_schedule);
   lua_setfield(lstate, -2, "schedule");
 
+  // vim.uv
+  luv_set_loop(lstate, &main_loop.uv);
+  luaopen_luv(lstate);
+  lua_setfield(lstate, -2, "uv");
```

Comment:
maybe it makes sense to name this `loop` instead of `uv`, since libuv is an implementation detail.